webdriver-spec.html

```diff
@@ -2191,7 +2191,7 @@
        in <var>matched capabilities</var> to <var>capability value</var>.
 
      <dt>"<code>pageLoadStrategy</code>"
-     <dd><p>If <var>capability value</var> is not a string,
+     <dd><p>If <var>capability value</var> is not in the <a>page load strategies table</a>,
```

Comment:
Why not just `in the <a>table of page load strategies</a>`. That would be equally good English and avoid the need for the alternative definition below.